src/w3d/lib/iniclass.cpp

```diff
@@ -446,7 +446,7 @@ double INIClass::Get_Double(const char *section, const char *entry, double defva
         // Is this value really a percentage? if so, divide it by 100.
         //
         if (strchr(entryptr->Get_Value(), '%')) {
-            return defvalue / 100.0;
+            return defvalue / 100.0f;
```

Comment:
This should probably not be changed given that its operating on a double value.

Reply:
Ups. Reverted.